apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPoolOptions.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+
+namespace Google.Cloud.Spanner.V1
+{
+    internal sealed class SessionPoolOptions
```

Comment:
yes, these were part of SessionPool earlier.  I've extracted them so I can pass them down to child objects without needing to pass the entire root object.

https://github.com/GoogleCloudPlatform/google-cloud-dotnet/pull/1176/files#diff-651f8cc55e7fee6255d710a72c806914